### ISSUE TITLE
commands not working at all

### ISSUE THREAD

**Issue by nightlifex** [gabyfle/gName-Changer#25]
all of the commands are not working

**Comment by nightlifex** [comment]
Correction:
only the !gname_save_all command does not work

**Comment by gabyfle** [comment]
Can you provide more info please ? Is there any errors ?

**Comment by nightlifex** [comment]
no there seems to be no errors, but when I use that command, no data positions are written to "garrysmod/data/gabyfle-rpname/npc_rpname_pos_rp_evocity2_v5p.txt"
the file should be written to with the data positions but the file is empty and has 0 kb file size

**Comment by gabyfle** [comment]
may be related to https://github.com/gabyfle/gName-Changer/issues/24

**Comment by nightlifex** [comment]
I do not have any issues with an infinite name change box, the name changing works fine and all other commands  as well

**Comment by gabyfle** [comment]
I understood but I’m pretty sure these two issues are related

De : nightlifex ***@***.***>
Envoyé : jeudi 22 avril 2021 19:11
À : gabyfle/gName-Changer ***@***.***>
Cc : Gabriel Santamaria ***@***.***>; Assign ***@***.***>
Objet : Re: [gabyfle/gName-Changer] commands not working at all (#25)


except the save npc command

**Comment by nightlifex** [comment]
Oh okay
Will you be able to fix the issue?
And could you tell me how I could manually write the coordinates in the npc datalist for spawning it every server restart

Have a great day!

**Comment by gabyfle** [comment]
Yes I’ll be able to fix it but I’m not sure when. This is a very old project and I do not have that much time to work on.
